### PR TITLE
Added support for multi episodes

### PR DESCRIPTION
**Pull request by SubJunk**
Closes https://github.com/UniversalMediaServer/api/issues/737

### CODE REVIEW

File: test/e2e/media-video.spec.ts [hunk 1]
```diff
@@ -242,10 +256,40 @@ describe('get by all', () => {
       expect(response.data.type).toEqual('episode');
       expect(response.data.seriesIMDbID).toEqual(EPISODE_PRISONBREAK.seriesIMDbID);
     });
+    // this also tests opensubtitles validation
+    test.only('should return two episodes when passed all possible params, from source APIs then store', async() => {
```
Comment (js-kyle) [inline test/e2e/media-video.spec.ts:260]:
```suggestion
    test('should return two episodes when passed all possible params, from source APIs then store', async() => {
```

File: package.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
     "purge": "TS_NODE_FILES=true ts-node --files scripts/purge",
     "start": "ts-node --files src/app.ts",
     "start:prod": "./node_modules/.bin/pm2 start ecosystem.config.js --update-env",
-    "test": "unset IMDB_API_KEY && OS_API_USERAGENT=OS_API_MOCK_KEY jest test/e2e/media-video.spec.ts --coverage --forceExit --detectOpenHandles && npm run test:mocks",
-    "test:mocks": "unset IMDB_API_KEY && OS_API_USERAGENT=OS_API_MOCK_KEY jest test/**/*.spec.ts  --testPathIgnorePatterns test/e2e/media-video.spec.ts --coverage --forceExit --detectOpenHandles",
+    "test": "unset IMDB_API_KEY && unset NODE_ENV && OS_API_USERAGENT=OS_API_MOCK_KEY jest test/e2e/media-video.spec.ts --coverage --forceExit --detectOpenHandles && npm run test:mocks",
```
Comment (js-kyle) [inline package.json:14]: How come this is needed? Jest injects NODE_ENV with `test`

**Review by js-kyle — COMMENTED** [review]
Approved once exclusive test is removed

File: package.json (hunk 2 above)
Comment (SubJunk) [reply package.json:14]: @js-kyle Jest only injects that value if `NODE_ENV` is not set already https://jestjs.io/docs/environment-variables#node_env

**Pull request merged**